Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
icustay_record_data = wfdb.rdheader(icustay_record_name, pn_dir=icustay_record_dir, rd_segments=False)
print("Done: metadata loaded for ICU stay '{}' from header file at URL: {}".format(icustay_record_name, "https://physionet.org/content/" + record_dir + record_name + ".hea"))
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Input [0;32mIn [6][0m, in [0;36m<cell line: 2>[0;34m()[0m
[1;32m      1[0m icustay_record_data [38;5;241m=[39m wfdb[38;5;241m.[39mrdheader(icustay_record_name, pn_dir[38;5;241m=[39micustay_record_dir, rd_segments[38;5;241m=[39m[38;5;28;01mFalse[39;00m)
[0;32m----> 2[0m [38;5;28mprint[39m([38;5;124m"[39m[38;5;124mDone: metadata loaded for ICU stay [39m[38;5;124m'[39m[38;5;132;01m{}[39;00m[38;5;124m'[39m[38;5;124m from header file at URL: [39m[38;5;132;01m{}[39;00m[38;5;124m"[39m[38;5;241m.[39mformat(icustay_record_name, [38;5;124m"[39m[38;5;124mhttps://physionet.org/content/[39m[38;5;124m"[39m [38;5;241m+[39m [43mrecord_dir[49m [38;5;241m+[39m record_name [38;5;241m+[39m [38;5;124m"[39m[38;5;124m.hea[39m[38;5;124m"[39m))

[0;31mNameError[0m: name 'record_dir' is not defined
NameError: name 'record_dir' is not defined

